src/AppInstallerCLIE2ETests/TestData/Manifests/TestMsixInstaller_WithSignatureHash.yaml

```diff
@@ -1,12 +1,12 @@
-Id: AppInstallerTest.TestMsixWithSignatureHash
-Name: TestMsixWithSignatureHash
-Version: 1.0.0.0
-Publisher: AppInstallerTest
-License: Test
-Installers:
-  - Arch: x86
-    Url: https://github.com/microsoft/appinstaller-cli/raw/master/src/AppInstallerCLIE2ETests/TestData/AppInstallerTestMsixInstaller.msix
-    Sha256: 714c861a52478bfa44228739c500c9429f4e75c7b78d48cac06bfb8f62bad627
-    SignatureSha256: d8a54b79a9c5956df5b338fdaf37f48e56316465defd182cbe64d8e5d3d53d4e
-    InstallerType: msix
-ManifestVersion: 0.1.0
+Id: AppInstallerTest.TestMsixWithSignatureHash
+Name: TestMsixWithSignatureHash
+Version: 1.0.0.0
+Publisher: AppInstallerTest
+License: Test
+Installers:
+  - Arch: x86
+    Url: https://localhost:5001/TestKit/x86/Release/AppInstallerCLIE2ETests/TestData/AppInstallerTestMsixInstaller/AppInstallerTestMsixInstaller.msix
+    Sha256: <MSIXHASH>
+    SignatureSha256: cc97d9b11bf271013a6c0de93da8983b6aa2505e70401bbe5bb33754b6e5d108

```

Comment:
>cc97d9b11bf271013a6c0de93da8983b6aa2505e70401bbe5bb33754b6e5d108 [](start = 21, length = 64)

This will need to be dynamically set as well.

Reply:
Created HashSignatureFromMSIX method to dynamically generate signature hash from msix installer file